withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -747,18 +749,27 @@ PROCEDURE READCONFIGFILESETTINGS
     END
   END
 
```

Comment:
Maybe wrap all config post processing below in a IF READCONFIGERROR=FALSE block since there is no need to continue processing if there was a read error.